pkg/arvo/sys/vane/gall.hoon

```diff
@@ -596,16 +601,27 @@
   ::  +mo-handle-sys-way: handle response to outgoing remote request
   ::
   ++  mo-handle-sys-way
-    |=  [=path =sign-arvo]
+    |=  [=wire =sign-arvo]
     ^+  mo-core
-    ::
-    ?>  ?=([%way @ @ @ ~] path)
-    =/  =ship           (slav %p i.t.path)
-    =/  foreign-agent   i.t.t.path
-    =/  remote-request  (remote-request i.t.t.t.path)
+    ?>  ?=([%way @ @ $@(~ [@ ~])] wire)
+    =/  =ship           (slav %p i.t.wire)
+    =/  foreign-agent   i.t.t.wire
     ::
     ?+    sign-arvo  !!
         [%a %done *]
+      =^  remote-request  outstanding.agents.state
+        ?~  t.t.t.wire
+          =/  stand
+            %+  ~(gut by outstanding.agents.state)  [sys+wire hen]
+            *(qeu remote-request)
+          ~|  [sys+wire=wire hen=hen stand=stand outs=outstanding.agents.state]
+          =^  rr  stand  ~(get to stand)
+          [rr (~(put by outstanding.agents.state) [wire hen] stand)]
+        ::  non-null case of wire is old, remove on next breach after
+        ::  2019/12
+        ::
+        [(remote-request i.t.t.t.wire) outstanding.agents.state]
```

Comment:
I think this should have a ;; on the call to +remote-request

Reply:
Done